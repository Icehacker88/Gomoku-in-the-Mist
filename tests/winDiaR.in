place A1
place B1
place B2
place C2
place C3
place D3
place D4
place E4
place E5
