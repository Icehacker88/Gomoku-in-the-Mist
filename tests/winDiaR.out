Black wins!
A1B1B2C2C3D3D4E4E5
Thank you for playing!
